BraintreeCore/src/main/java/com/braintreepayments/api/GraphQLConstants.kt

```diff
@@ -0,0 +1,48 @@
+package com.braintreepayments.api
+
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+object GraphQLConstants {
```

Comment:
This top-level `@RestrictTo` annotation should be the only one we need. We can delete the others.

Reply:
```suggestion
/**
 * @suppress
**/
@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
object GraphQLConstants {
```
We should suppress `@RestrictTo` annotated classes in the docs too.